src/BlazorDB/BlazorDbFactory.cs

```diff
@@ -40,11 +45,51 @@ async Task BuildFromServices()
                 foreach(var dbStore in dbStores)
                 {
                     Console.WriteLine($"{dbStore.Name}{dbStore.Version}{dbStore.StoreSchemas.Count}");
-                    var db = new IndexedDbManager(dbStore, _jsRuntime);
+                    var db = new IndexedDbManager(dbStore, _jsRuntime, _dbs);
                     await db.OpenDb();
                     _dbs.Add(dbStore.Name, db);
                 }
             }
         }
+
+        async Task BuildExistingDynamic(string dbName)
+        {
+            var dbStore = await GetDbStoreDynamic(dbName);
+            var db = new IndexedDbManager(dbStore, _jsRuntime, _dbs);
+            _dbs.Add(dbStore.Name, db);
+        }
+
+        async Task BuildNewDynamic(string dbName)
+        {
+            var newDbStore = new DbStore() { Name = dbName, StoreSchemas = new List<StoreSchema>(), Version = 0, Dynamic = true };
```

Comment:
Rather than open the DB here and have your logic in the javascript version to get around the Dexie bug, should we not allowed a DB to be opened if there is no schema?  What's the point to open it anyway?

Thinking out loud, maybe in `AddSchema` we detect if any previous schemas and then open after at least 1 schema is added?